src/main/java/com/kammatysiak/medicalclinic/model/entity/Visit.java

```diff
@@ -0,0 +1,43 @@
+package com.kammatysiak.medicalclinic.model.entity;
+
+import com.kammatysiak.medicalclinic.model.dto.VisitDTO;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "VISITS")
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
```

Comment:
W entity inaczej powinien wygladac hashcode i equals 
https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/